theseus/utils/examples/__init__.py

```diff
@@ -50,4 +50,10 @@
     )
 
 
-from .pose_graph import PosePriorError, RelativePoseError
+from .pose_graph import (
+    PoseGraphDataset,
+    PoseGraphEdge,
+    PosePriorError,
+    RelativePoseError,
```

Comment:
We can get rid of these (and their files) now since we have moved to using Difference and Between, right?

Reply:
Yes, they are equivalent to `Difference` and `Between`.